animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -88,6 +89,13 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
             self.newInput("Float", "End Size", "endSize", value = 0.5, minValue = 0)
             self.newInput("Float", "Start Angle", "startAngle", value = 0)
             self.newInput("Float", "End Angle", "endAngel", value = 6 * PI)
+        elif self.mode == "FIBONACCI":
+            self.newInput("Integer", "Amount", "amount", value = 100, minValue = 1)
+            self.newInput("Integer", "Center mask", "centermask",value = 0, minValue = 0)
```

Comment:
Change `Center mask -> Center Mask`, and `centermask -> centerMask`. Do the same for other variables.